regolith/helpers/u_milestonehelper.py

```diff
@@ -272,8 +270,16 @@ def db_updater(self):
                         doc.update({'kickoff': pdoc})
                     rc.client.update_one(rc.database, rc.coll, {'_id': id[0]}, doc)
                     if not rc.finish:
-                        print(f"The milestone uuid {uuid} in {id[0]} has been updated in projecta.")
+                        success.append(f"The milestone uuid {uuid} in {id[0]} has been updated in projecta.")
                 else:
-                    print(f"Multiple ids match your entry ({uuid}).\n"
-                          "Try entering six or more characters of the uuid and rerunning the helper.")
+                    fail.append(f"Multiple ids match your entry ({uuid}).\n"
+                          "Try entering six or more characters of the uuid and rerunning the helper "
+                          f"to update milestone ({uuid}).")
+        if success:
```

Comment:
This is basically it but the logic breaks here because, in principle, you could have success and failure but that is not possible.

Maybe something like:
```
print(f"{*fail,}") if fail else print(f"{*success,}")
```
I am not sure if this will give the right behavior, but we can try.  

Also, I think there may be a logic problem higher up that it can still update a database that doesn't contain the prum under some circumstance (I found it when finishing a milestone).  This actually may be the main problem, that it succeeds when it should fail.

Reply:
I think the problem is the logic as well. I am not sure we can use the updater to update multiple milestones without an `update_many` function like the one in pymongo or mongo query language. We only have the `update_one` to work with right now. I think it will be best for me to remove the loop that allows the user to input multiple uuids. The way the code is written allows for success and failure in instances in which some milestone uuids are located in a database and others are not. Since most users are updating one milestone at a time, I do not think it is a major hindrance to take remove this functionality until we can utilize the `update_many` function. The `update_many` function will allow us to make sure there are no errors with any of the ids or milestones before the helper updates the database.